lib/index.js

```diff
@@ -152,8 +152,7 @@ export default {
         const cmd = [this.executablePath, 'analyze',
           '-f json',
           makeEngineString(linterEnabledEngines),
-          `'${atom.project.relativize(filePath)}'`,
-          '< /dev/null'].join(' ');
```

Comment:
@maxjacobson You're right, `< /dev/null` stuff didn't make it fail. I've just checked it locally. It all had to do with the timeout issue, but I changed both at the same time and that was my fault. Sorry. Reverting.